server/apps/ldap/ldap.py

```diff
@@ -194,7 +197,7 @@ class ImportUserProfileService(UsersService):
     def on_create(self, docs):
         for index, doc in enumerate(docs):
             # ensuring the that logged in user is importing the profile.
-            if flask.g.user.get('username') != doc.get('username'):
+            if get_user().get('username') != doc.get('username'):
```

Comment:
Should the `get_user()` call be moved outside the loop? It doesn't seem to depend on any iteration variables...